study-builder/studies/basil/picklist-autocomplete-activity.conf

```diff
@@ -0,0 +1,31 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": ${id.study},
+  "activityCode": ${id.act.autocomplete},
+  "versionTag": "v1",
+  "displayOrder": 2,
+  "writeOnce": false,
+  "editTimeoutSec": null,
+  "maxInstancesPerUser": 5,
```

Comment:
Let's remove `maxInstancesPerUser` so we can create as many of these as we want for testing.

Reply:
Fixed